src/styles/main.scss

```diff
@@ -1,3 +1,65 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
 }
+
+$day-size: 100px;
+$calendar-gap: 1px;
+$calendar-padding: 10px;
+$calendar-width: 2 * $calendar-padding + 7 * $day-size + 6 * $calendar-gap;
+$start-day: 'mon' 0,
+  'tue' 1,
+  'wed' 2,
+  'thu' 3,
+  'fri' 4,
+  'sat' 5,
+  'sun' 6;
+
+.calendar {
```

Comment:
move calendar styles to `calendar.scss` file.
And when you have 3 files (variables, common, calendar) import them to `index.scss`. This file you will import to `html`